.circleci/config.yml

```diff
@@ -40,6 +40,7 @@ test_containers:
     DD_AGENT_HOST: testagent
     DD_TRACE_AGENT_PORT: 9126
     DATADOG_GEM_CI: true
+    RUBYOPT: -r/app/.circleci/patch_log_device
```

Comment:
I would suggest applying this in `spec_helper.rb`, as we have a few other similar tweaks for the test suite there; applying it via RUBYOPT may collide with other things we may be testing in the test suite. (Because it affects every ruby process that gets run, including e.g. bundler and whatnot)